src/components/expanded-state/custom-gas/FeesGweiInput.js

```diff
@@ -135,7 +135,6 @@ export default function FeesGweiInput({
         color={buttonColor}
         onBlur={onBlur}
         onChange={onChange}
-        onFocus={onInputPress}
```

Comment:
this was also causing many calls to the input ref focus and messing up the magic focus